tensorflow/compiler/mlir/tools/kernel_gen/kernel_creator.cc

```diff
@@ -220,10 +221,17 @@ Status LowerTFToJITInvocation(mlir::ModuleOp module,
 
 Status LowerTFtoLoops(mlir::ModuleOp module, llvm::ArrayRef<int64_t> tile_sizes,
                       llvm::ArrayRef<int64_t> unroll_factors,
-                      int64_t max_supported_rank, bool cpu_codegen) {
+                      int64_t max_supported_rank, bool cpu_codegen,
+                      bool jit_i64_indexed_for_large_tensors) {
   mlir::PassManager pm(module.getContext());
   applyTensorflowAndCLOptions(pm);
-
+  if(jit_i64_indexed_for_large_tensors)
+  {
+    pm.addNestedPass<mlir::FuncOp>(
+      mlir::kernel_gen::transforms::CreateTFToJITInvocationPass(
+          architectures, tile_sizes, unroll_factors, max_supported_rank,
+          cpu_codegen, true));
```

Comment:
Can we have a comment for the `true`? 

`/*jit_i64_indexed_for_large_tensors=*/ true`

Reply:
comment added.